sys/include/net/conn.h

```diff
@@ -15,6 +15,7 @@
  * About
  * =====
  *
+ * @code
```

Comment:
could you use [markdown fences](https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_fenced) instead, please?